docs/spec/governance/tags.md

```diff
@@ -0,0 +1,38 @@
+# Tags
+
+The governance module emits the following events/tags:
+
+## EndBlocker
```

Comment:
What does this mean? All governance events are tagged with these?

Reply:
Exactly.